Dockerfile

```diff
@@ -46,9 +46,10 @@ RUN apt-get -qq update \
   && rm /tmp/sym-collector.zip \
   && chmod +x /usr/local/bin/SymbolCollector.Console
 
-RUN curl -fsSL https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_3.7.1-stable.tar.xz -o /opt/flutter_linux_3.7.1-stable.tar.xz \
-  && tar xf /opt/flutter_linux_3.7.1-stable.tar.xz -C /opt \
-  && rm /opt/flutter_linux_3.7.1-stable.tar.xz
+# https://docs.flutter.dev/get-started/install/linux#install-flutter-manually
+RUN curl -fsSL https://storage.googleapis.com/flutter_infra_release/releases/stable/linux/flutter_linux_3.10.0-stable.tar.xz -o /opt/flutter_linux_3.10.0-stable.tar.xz \
+  && tar xf /opt/flutter_linux_3.10.0-stable.tar.xz -C /opt \
+  && rm /opt/flutter_linux_3.10.0-stable.tar.xz
```

Comment:
to make this less diff you can `-o flutter.tar.xz` and use that elsewhere so that only the version number changes each time

Reply:
[`deceae0` (#463)](https://github.com/getsentry/craft/pull/463/commits/deceae0696e58e1efbe5dc2973fa2a17e4907919)